player/lua/select.lua

```diff
@@ -18,6 +18,13 @@ License along with mpv.  If not, see <http://www.gnu.org/licenses/>.
 local utils = require "mp.utils"
 local input = require "mp.input"
 
+local options = {
+    save_watch_history = false,
+    watch_history_path = "~~state/watch_history.txt",
```

Comment:
For symmetry with `watch_later`, maybe don't use `.txt`?

Reply:
`watch_later` is a directory?

Reply:
Oh you meant the files inside `watch_later`. Dunno I think those are a weird case. I think the txt extension makes it easy to open it in a text editor by default e.g. in Windows or with zsh suffix aliases which are what I use. I also used `console_history.txt` in the other PR so that would have to be changed too.

Reply:
ok, maybe use `.json` or `.jsonl` in this case?

Reply:
Well as I wrote in the commit message I didn't use jsonl only because I can't figure out whether ndjson or jsonl is preferable. Will switch to jsonl.